syntax = "proto3";
import "nanopb.proto";

message SecureData {
  string sender = 1 [(nanopb).max_size = 65];
  string payload = 2 [(nanopb).max_size = 256];
}